iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/TopologyModificationUtils.java

```diff
@@ -171,6 +200,71 @@ private static <L extends LoadingLimits, A extends LoadingLimitsAdder<L, A>> voi
+        List<TemporaryLimitsBag> temporaryLimitsBags = new ArrayList<>();
+        Map<Integer, TemporaryLimitsBag> temporaryLimitsByAcceptableDuration = new HashMap<>();
+        double minPermanentLimitValue = Double.MAX_VALUE;
+
+        for (LoadingLimits loadingLimit : loadingLimits) {
+            // find the min permanent limit value
+            if (loadingLimit.getPermanentLimit() < minPermanentLimitValue) {
+                minPermanentLimitValue = loadingLimit.getPermanentLimit();
+            }
+
+            // find the min temporary limit value for each acceptableDuration
+            Collection<LoadingLimits.TemporaryLimit> temporaryLimits = loadingLimit.getTemporaryLimits();
+            for (LoadingLimits.TemporaryLimit temporaryLimit : temporaryLimits) {
+                Integer acceptableDuration = temporaryLimit.getAcceptableDuration();
+                if (!temporaryLimitsByAcceptableDuration.containsKey(acceptableDuration)) {
+                    temporaryLimitsByAcceptableDuration.put(acceptableDuration, new TemporaryLimitsBag(temporaryLimit));
+                }
+                if (temporaryLimit.getValue() < temporaryLimitsByAcceptableDuration.get(acceptableDuration).getValue()) {
+                    temporaryLimitsByAcceptableDuration.put(acceptableDuration, new TemporaryLimitsBag(temporaryLimit));
+                }
+            }
+            temporaryLimitsBags.addAll(temporaryLimitsByAcceptableDuration.values());
+        }
+
+        return new LoadingLimitsBag(minPermanentLimitValue, temporaryLimitsBags);
+    }
+
+    static LoadingLimitsBags calcMinLoadingLimitsBags(Line line1, Line line2) {
+        List<LoadingLimits> currentLimits = Stream.of(line1.getCurrentLimits1(), line1.getCurrentLimits2(), line2.getCurrentLimits1(), line2.getCurrentLimits2())
+                .filter(Optional::isPresent)
+                .map(Optional::get)
+                .collect(Collectors.toList());
+
+        List<LoadingLimits> activePowerLimits = Stream.of(line1.getActivePowerLimits1(), line1.getActivePowerLimits2(), line2.getActivePowerLimits1(), line2.getActivePowerLimits2())
+                .filter(Optional::isPresent)
+                .map(Optional::get)
+                .collect(Collectors.toList());
+
+        List<LoadingLimits> apparentPowerLimits = Stream.of(line1.getApparentPowerLimits1(), line1.getApparentPowerLimits2(), line2.getApparentPowerLimits1(), line2.getApparentPowerLimits2())
+                .filter(Optional::isPresent)
+                .map(Optional::get)
+                .collect(Collectors.toList());
+
+        return new LoadingLimitsBags(!activePowerLimits.isEmpty() ? calcMinLoadingLimitsBag(activePowerLimits) : null,
+                                     !apparentPowerLimits.isEmpty() ? calcMinLoadingLimitsBag(apparentPowerLimits) : null,
+                                     !currentLimits.isEmpty() ? calcMinLoadingLimitsBag(currentLimits) : null);
```

Comment:
Not really sure what you're doing here.
I think we should do it simple: we keep the limits of Side ONE of line1 and of Side TWO of line2 without changes. The other limits are lost (maybe with a log warn to indicate it).

Reply:
It was specified like that in the US https://dev.azure.com/GridSuite/MVP/_sprints/backlog/MVP%20Team/MVP/Sprint%2051?workitem=1190 :

seuils = min(seuils des liaisons remplacées)

Reply:
The issue is it might mean loss of information in some cases... Maybe we should make the strategy of merging of limits a parameter: something like `BiFunction<Line, Line, LoadingLimitsBags> getLimits1, BiFunction<Line, Line, LoadingLimitsBags> getLimitts2`?

Reply:
VL1 (OL11) --------------- (OL12) tee point (OL21)--------------------(OL22) VL2
OL for Operational Limits. We think that we have to:
- First, you have to check that you have initially OL12 and OL21 are present. Ideally you have to distinguish if you have only a permanent limit or more.
- If nothing is present, we just leave OL11 and OL22 where they are.
- If OL12 have a permanent limit, you can merge it with OL11 using the minimum if needed. 
- If OL12 have temporary limits and also have OL11, for me they are lost with a warning/report. If OL11 has no temporary limits, you can move it on OL11.
- If OL21 have a permanent limit, you can merge it with OL22 using the minimum if needed. If OL22 has no temporary limits, you can move it on OL22.

Reply:
Do I have to implement that in this PR ?

Reply:
Modifications done